### PR TITLE
Added background colour to webpage

### PR DESCRIPTION
**Pull request by Gregamoo**
Added material ui and changed background

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -6,4 +6,7 @@ module.exports = {
     "^.+\\.(js|jsx)$": "babel-jest",
   },
   testEnvironment: "jsdom",
+  moduleNameMapper: {
+    "\\.(css|less)$": "<rootDir>/src/mocks/styleMock.js",
```
Comment (emilybuc) [inline jest.config.js:10]: Why did you change this?

File: src/App.test.js [hunk 2]
```diff
@@ -3,6 +3,6 @@ import App from "./App";
 
 test("fuji digital h1 renders on page", () => {
   render(<App />);
-  const title = screen.getByText(/Fuji Digital./i);
-  expect(title).toBeInTheDocument();
```
Comment (emilybuc) [inline src/App.test.js:7]: we should definitely add a new test back in here on the next PR

File: src/index.js [hunk 3]
```diff
@@ -2,10 +2,12 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
+import CssBaseline from "@mui/material/CssBaseline";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
+    <CssBaseline />
```
Comment (emilybuc) [inline src/index.js:10]: What is this?

File: jest.config.js (hunk 1 above)
Comment (Gregamoo) [reply jest.config.js:10]: jest wouldnt render css file so had to replace with default

File: src/index.js (hunk 3 above)
Comment (Gregamoo) [reply src/index.js:10]: This is the same as normalize but using the mui

**Review by emilybuc — APPROVED** [review]
